test/rhsmlib/facts/test_dmifacts.py

```diff
@@ -0,0 +1,109 @@
+# Copyright (c) 2022 Red Hat, Inc.
+#
+# This software is licensed to you under the GNU General Public
+# License as published by the Free Software Foundation; either version
+# 2 of the License (GPLv2) or (at your option) any later version.
+# There is NO WARRANTY for this software, express or implied,
+# including the implied warranties of MERCHANTABILITY,
+# NON-INFRINGEMENT, or FITNESS FOR A PARTICULAR PURPOSE. You should
+# have received a copy of GPLv2 along with this software; if not, see
+# http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+#
+
+from rhsmlib.facts.dmidecodeparser import DmidecodeParser
+from rhsmlib.facts.dmiinfo import DmidecodeFactCollector
+
+import contextlib
+import os
+import unittest
+
+
+FAKE_PART_NUMBER = "AAAAAAAAAAAAA-ZZ"
+FAKE_PRODUCT_NAME = "PPPPPPPPPP"
```

Comment:
`FAKE_PART_NUMBER` and  `FAKE_PRODUCT_NAME` are not used for any assertion despite I can see these values in testing .txt files.

Reply:
I wanted to use them for testing, however they are even less available than e.g. serial number :/

I left them there with the used ones because they are all data "anonymized" in the test data.

Reply:
I can see that testing files are anonymized. If you are not going to use these variables for testing, then it doesn't make much sense to keep these variables.